pkg/operator/ceph/agent/agent.go

```diff
@@ -85,6 +87,16 @@ func (a *Agent) createAgentDaemonSet(namespace, agentImage, serviceAccount strin
 		return fmt.Errorf("invalid agent mount security mode specified (given: %s)", agentMountSecurityMode)
 	}
 
+	rookEnableSelinuxRelabeling := os.Getenv(RookEnableSelinuxRelabelingEnv)
```

Comment:
You could skip all the error checking here and just pass `os.Getenv(RookEnableSelinuxRelabelingEnv)` for the value on line 147. The error checking can be delayed until the `init.go`.